### PR TITLE
Improve a sentence in transactions developer docs (Chinese)

### PR DESCRIPTION
**Pull request by hyper0x**
improve a sentence

### CODE REVIEW

**Review by minimalsm — COMMENTED** [review]
Thanks for the PR @hyper0x, will get a review from a native speaker before we bring this in 💪

**Comment by corwintines** [comment]
@minimalsm do we have any updates on this?

File: src/content/translations/zh/developers/docs/transactions/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ preMergeBanner: true
 
 ## 什么是交易？ {#whats-a-transaction}
 
-以太坊交易是指由外部持有账户发起的行动，换句话说，是指由人管理而不是智能合约管理的账户。 例如，如果 Bob 发送 Alice 1 ETH，则 Bob 的帐户必须减少 1 ETH，而 Alice 的账户必须增加 1 ETH。 此项操作发生在交易中，会变更状态。
+以太坊交易是指由外部持有账户发起的行动，换句话说，是指由人管理而不是智能合约管理的账户。 例如，如果 Bob 向 Alice 发送 1 ETH，则 Bob 的帐户必须减少 1 ETH，而 Alice 的账户必须增加 1 ETH。 此项操作发生在交易中，会变更状态。
```
Comment (minimalsm) [inline src/content/translations/zh/developers/docs/transactions/index.md:17]:
```suggestion
以太坊交易是指由外部持有账户发起的行动，换句话说，是指由人管理而不是智能合约管理的账户。 例如，如果 Bob 向 Alice 发送 1 ETH，则 Bob 的帐户必须减少 1 ETH，而 Alice 的账户必须增加 1 ETH。 交易会造成状态的改变。
```

**Review by minimalsm — APPROVED** [review]
Sorry for the delay, and thanks @hyper0x for the PR ❤️

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @hyper0x for content, translations

**Pull request merged**